src/components/Button/Button.tsx

```diff
@@ -17,60 +17,37 @@ const Button = forwardRef(
     { children, variant, size, icon, iconAlignment = 'left', iconSize, className, ...buttonAttrs }: ButtonProps,
     ref: ForwardedRef<HTMLButtonElement>
   ): JSX.Element => {
-    let classes = '';
-    switch (variant) {
-      case 'assertive':
-        classes = 'bsds-button-assertive';
-        break;
-      case 'ghost':
-        classes = 'bsds-button-ghost';
-        break;
-      case 'subtle':
-        classes = 'bsds-button-subtle';
-        break;
-      default:
-        classes = 'bsds-button';
-        break;
-    }
+    const buttonClasses = classNames(
```

Comment:
let's avoid refactoring in the context of this PR. if we want to move forward with the classname package, we'll want to refactor all components to follow the same patterns.

Reply:
Sorry, I didn't do a good job explaining here. This branch and PR were created while working on https://github.com/bos-sci/anatomy-react/pull/73. The refactor was meant to demo what could be done with the addition of the `classnames` package.

After #73 is merged there will be fewer changes in this branch compared to develop.